pkg/kyverno/apply/command.go

```diff
@@ -58,7 +58,7 @@ func Command() *cobra.Command {
 			}()
 
 			if len(resourcePaths) == 0 && !cluster {
-				return sanitizedError.New(fmt.Sprintf("Specify path to resource file or cluster name"))
+				return sanitizedError.NewWithError(fmt.Sprintf("Specify path to resource file or cluster name"), err)
```

Comment:
change to, "resource file or cluster required "

Reply:
updated accordingly